### PR TITLE
command: return non-zero exit code for errors and add ignore-glacier-warnings option

### LINKED ISSUE #304: Some errors do not result in a non-zero exit code
## System info

**OS:** Alpine Linux 3.13.1
**s5cmd version:** v1.2.1-3fc01af

## Observations

The host is resolvable and reachable, but credentials are not passed through envs:

```shell
/ $ s5cmd --log error --endpoint-url https://minio.localhost cp "s3://tmp/*" tmp/
ERROR session: fetching region failed: NoCredentialProviders: no valid providers in chain. Deprecated.
	For verbose messaging see aws.Config.CredentialsChainVerboseErrors
ERROR "cp s3://tmp/* tmp/": NoCredentialProviders: no valid providers in chain. Deprecated. For verbose messaging see aws.Config.CredentialsChainVerboseErrors
/ $ echo $?
0
```

The host was resolvable, unreachable:
```shell
/ $ s5cmd --log error --endpoint-url https://minio.localhost cp "s3://tmp/*" tmp/
ERROR session: fetching region failed: RequestError: send request failed
caused by: dial tcp 255.255.255.254:443: connect: connection refused
ERROR "cp s3://tmp/* tmp/": RequestError: send request failed caused by: Get "https://minio.localhost/tmp?list-type=2&prefix=": dial tcp 255.255.255.254:443: connect: connection refused
/ $ echo $?
0
```

The host is unresolvable:
```shell
/ $ s5cmd --log error --endpoint-url https://bvsdfdjsfs cp "s3://tmp/*" tmp/
ERROR session: fetching region failed: RequestError: send request failed
caused by: dial tcp: lookup bvsdfdjsfs on 127.0.0.1:53: no such host
ERROR "cp s3://tmp/* tmp/": RequestError: send request failed caused by: Get "https://bvsdfdjsfs/tmp?list-type=2&prefix=": dial tcp: lookup bvsdfdjsfs on 127.0.0.1:53: no such host
/ $ echo $?
0
```

All of that means that we can't be sure that pipelines fail in case there's something wrong with `s3cmd cp`.

### LINKED ISSUE #346: Ignore Glacier warnings
After https://github.com/peak/s5cmd/commit/9defce6b615f10bd2be785ca42cf510ce3642bf8, commands have started to return exit status 1 if glacier warnings are encountered during execution. It is a breaking change for users who use version v1.3.0.  I think it would be useful to add a flag for turning on/off glacier warnings similar to ([--ignore-glacier-warnings](https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3/cp.html.))). As default, commands should exit with 0 for glacier warnings if we don't wanna break current API (v1.3.0).

### PR DESCRIPTION
**Pull request by sonmezonur**
Fixes #304, fixes #346 (which are release-blockers for v2.0.0)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,8 +2,12 @@
 
 ## not released yet
 
+#### Features
+- Added `--ignore-glacier-warnings` option to `cp` and `select` commands. ([#346](https://github.com/peak/s5cmd/issues/346))
```
Comment (igungor) [inline CHANGELOG.md:6]:
```suggestion
- Added `--ignore-glacier-warnings` option to `cp`, `mv`, and `select` commands. ([#346](https://github.com/peak/s5cmd/issues/346))
```

Comment (igungor) [reply CHANGELOG.md:6]: Also this PR adds `force-glacier transfer` flag to the `select` command.

File: command/cp.go [hunk 2]
```diff
@@ -162,6 +162,10 @@ func NewCopyCommandFlags() []cli.Flag {
 			Name:  "force-glacier-transfer",
 			Usage: "force transfer of GLACIER objects whether they are restored or not",
```
Comment (igungor) [inline command/cp.go:163]:
```suggestion
			Usage: "force transfer of glacier objects whether they are restored or not",
```

File: command/select.go [hunk 3]
```diff
@@ -56,6 +56,14 @@ func NewSelectCommand() *cli.Command {
 				Name:  "exclude",
 				Usage: "exclude objects with given pattern",
 			},
+			&cli.BoolFlag{
+				Name:  "force-glacier-transfer",
+				Usage: "force transfer of GLACIER objects whether they are restored or not",
```
Comment (igungor) [inline command/select.go:61]:
```suggestion
				Usage: "force transfer of glacier objects whether they are restored or not",
```

**Comment by igungor** [comment]
> Fixes #304 and #346 (which are release-blockers for v2.0.0)

Could you adjust the PR description according to https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword.

**Comment by sonmezonur** [comment]
> > Fixes #304 and #346 (which are release-blockers for v2.0.0)
> 
> Could you adjust the PR description according to https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword.

done

File: CHANGELOG.md (hunk 1 above)
Comment (sonmezonur) [reply CHANGELOG.md]: done

**Review by igungor — APPROVED** [review]

**Comment by sonmezonur** [comment]
@ilkinulas PTAL

**Review by ilkinulas — APPROVED** [review]

**Pull request merged**